__tests__/src/getPropValue-test.js

```diff
@@ -357,17 +359,21 @@ describe('getPropValue', () => {
     });
 
     it('should evaluate to a correct representation of member expression with a nullable member', () => {
-      // This tell will not throw when Babel is upgraded from 6 to 7. Remove
-      // the throw expectation wrapper at that time.
-      // eslint-disable-next-line no-undef
-      expect(() => {
+      const runTest = () => {
```

Comment:
Here I have extracted the expected-to-fail function into a variable to conditionally expect it to fail or not according to Babel version